RenderStatePackager/src/RenderStatePackager.cpp

```diff
@@ -223,7 +225,7 @@ bool RenderStatePackager::Execute(RefCntAutoPtr<IArchiver> pArchive)
                         PipelineCI.pAS = FindShader(pPipelineDescRSN->pASName);
                         PipelineCI.pMS = FindShader(pPipelineDescRSN->pMSName);
 
-                        if (!pArchive->AddGraphicsPipelineState(PipelineCI, PipelineStateArchiveInfo{PSO_ARCHIVE_FLAG_NONE, m_DeviceBits}))
+                        if (!pArchive->AddGraphicsPipelineState(PipelineCI, PipelineStateArchiveInfo{m_ArchiveFlags, m_DeviceFlags}))
```

Comment:
Duplicated 4 times.
Please init the ArchiveInfo struct once and use it with all `Add*PipelienState` calls